.travis.yml

```diff
@@ -3,31 +3,53 @@ language: julia
 os:
   - linux
   - osx
+  - windows
 julia:
-  - 1.0
-  - 1.1
-  - 1.2
+  - 1.0  # LTS
+  - 1.3  # Latest release
```

Comment:
I'll change this to `1` so this is automatic

Reply:
I've rolled this back as although this is handy we may not want stop testing on the old release right away